.github/workflows/sync.sh

```diff
@@ -9,6 +9,16 @@ tests="types/google.maps/google.maps-tests.ts"
 # Run from root of workspace
 cd "${GITHUB_WORKSPACE}"
 
+# Extract the version number from js-types/VERSION
```

Comment:
This could just be a single command:

version=$(cat js-types/VERSION | cut -d '.' -f1-2)

Reply:
fixed